crates/nu-cli/src/cli.rs

```diff
@@ -283,6 +283,11 @@ pub fn create_default_context(
             whole_stream_command(UrlQuery),
         ]);
 
+        #[cfg(unix)]
```

Comment:
The core team chatted and we decided that commands should exist on ALL platforms. That's core to the nushell philosophy.

Of course, as you've stated in a comment, `exec` doesn't work on Windows. We should simply emit an error on Windows saying that `exec` is not supported on the platform.

Reply:
Ok I can remove the cfg check, should I just `panic!` in windows?

Reply:
Nope, it would be good to return an `Err` with a message indicating `exec` isn't available on the platform. Something like this in `exec.rs`:

```
Err(ShellError::labeled_error(
    "The exec command is not supported on this platform",
    "unavailable for this platform",
    &name_tag,
))
```

Reply:
☝️ which you can do with a `not(unix)` implementation for exec whose body is basically just returning that error 🙂